### PR TITLE
RES-492: Add object classification method

### PR DESCRIPTION
**Pull request by lscheinkman**
@subutai Please review

### CODE REVIEW

File: htmresearch/frameworks/layers/l2_l4_inference.py [hunk 1]
```diff
@@ -578,6 +580,42 @@ def getL2Representations(self):
...
+      score = 0.0
+      for i in xrange(self.numColumns):
+        # Ignore inactive column
+        if len(l2sdr[i]) == 0:
+          continue
+
+        count += 1
+        overlap = len(l2sdr[i] & objectSdr[i])
+        if overlap >= minOverlap:
+          score += overlap
```
Comment (subutai) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: I think this should just be += 1. The number of bits doesn't matter once it's above a threshold.

File: htmresearch/frameworks/layers/l2_l4_inference.py [hunk 2]
```diff
@@ -578,6 +580,42 @@ def getL2Representations(self):
...
+
+      if count == 0:
+        results[objectName] = 0
+      else:
+        results[objectName] = score / (count * sdrSize)
```
Comment (subutai) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: Just divide by `count`

File: htmresearch/frameworks/layers/l2_l4_inference.py [hunk 3]
```diff
@@ -578,6 +580,42 @@ def getL2Representations(self):
     return [set(column._pooler.getActiveCells()) for column in self.L2Columns]
 
 
+  def getCurrentClassification(self, minOverlap=None):
+    """
+    A dict with a score for each object. Score goes from 0 to 1. A 1 means
```
Comment (subutai) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: Should we return a numpy array, or would that make it hard for Unity?

Comment (lscheinkman) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: A dictionary is easier to convert to JSON for Unity

**Pull request merged**